KD_Lib/common/base_class.py

```diff
@@ -76,15 +103,29 @@ def train_student(self, epochs=10, plot_losses=True):
                 
                 loss = self.calculate_kd_loss(student_out, teacher_out, label)
 
+                if isinstance(student_out, tuple):
+                    student_out = student_out[0]
+
+                pred = student_out.argmax(dim=1, keepdim=True)
+                correct += pred.eq(label.view_as(pred)).sum().item()
+
                 self.optimizer_student.zero_grad()
                 loss.backward()
                 self.optimizer_student.step()
 
                 epoch_loss += loss
```

Comment:
If we could continuously return (or yield?) training losses, that should help a lot in analysis (say continuous graphs). Just an idea, what do you say?

Reply:
This will come in with the tensorboard support have an issue for that.

Reply:
:+1: that's a lot cooler then.